Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:14.04
+RUN apt-get update && apt-get install -y \
+    python-qt4 \
+    python-pyside \
+    python3-pyqt4 \
+    python3-pyside \
+    python-pip
+RUN pip install nose
+ENTRYPOINT nosetests --verbose /Qt.py/tests.py
```

Comment:
That's nice. Even shorter than what I had in mind. :)

```bash
WORKDIR /Qt.py
```

In which case we would still need to specify `nose` and things via the command-line.